### PR TITLE
Awesome Books: Website navigation

### PR DESCRIPTION
**Pull request by smunozmo**
# Documentation

All the documentation is on the README file.

## Updates:

*  Add navigation to the website using Javascript.
* Use logo to display everything.

## Notes:

Coded by @afcastaneda223 and @smunozmo 

✅  - No linters errors detected. 
✅  - Used Javascript for functions. 
✅  - Booklist created dynamically.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,6 +14,29 @@
 </head>
 
 <body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```
Comment (smcommits) [inline index.html:19]: - [ ] **Please make sure that the current date is displayed below the navigation bar.**

This is a project requirement and is also included in the wireframe. Please make sure to use the Luxon library.

image

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @smunozmo, 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Please check the inline comments below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (smunozmo) [reply index.html:19]: Hi @smcommits, how are you? 
I'm using this luxon library but it fails ESLinters, it more than 1000 errors, how can I sort this out?

Thank you.

Comment (smunozmo) [reply index.html:19]: I added luxon.js to .eslintignore to avoid those errors.

**Review by umararshadbut — APPROVED** [review]
Hi @afcastaneda223 and @smunozmo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
HAPPY CODING :computer:
**Umair Arshad**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by smunozmo** [comment]
Thank you guys! @smcommits @umararshadbut

**Pull request merged**